pkg/timeutils/utils_test.go

```diff
@@ -23,12 +26,17 @@ func TestGetTimestamp(t *testing.T) {
 		// unix timestamps returned as is
 		{"1136073600", "1136073600"},
 
+		// Durations
+		{"1m", fmt.Sprintf("%d", now.Add(-1*time.Minute).Unix())},
+		{"1.5h", fmt.Sprintf("%d", now.Add(-90*time.Minute).Unix())},
```

Comment:
Does this need a negative test case as well, e.g. `"1,5h"`?

(I'm not really reviewing, just browsing out of interest)